lib/splitio-api-binding.js

```diff
@@ -98,8 +98,45 @@ class SplitioApiBinding {
   /**
    * Get segment data.
    */
-  getSegments () {
-    throw new Error('Not implemented')
+  async getSegments ({ splits }) {
+    const allSegmentNames = []
```

Comment:
Given that this doesn't have duplicate values, should we consider making this a Set?